src/entity/common.ts

```diff
@@ -0,0 +1,41 @@
+export enum ReportEntity {
```

Comment:
Also here, we not fixing comment reports?

Reply:
Should be updated now.